src/vs/base/common/actionWidget/actionWidget.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IDisposable } from 'vs/base/common/lifecycle';
+
+export interface ActionSet<T> extends IDisposable {
```

Comment:
It's a bit all over the place whether the I prefix is used, because of that should we change to use it everywhere in the action widget code?

Reply:
yeah I agree